catalog/spatial/csw/spatial-csw-source/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/source/TestCswSource.java

```diff
@@ -962,6 +965,124 @@ public void testRefresh() throws SecurityServiceException {
     }
 
     @Test
+    public void testRefreshWithRegisterForEvents() throws Exception {
+        String subscriptionId = "subscriptionid";
```

Comment:
it looked ugly using too many parameters and I don't want to move this into the before method since only 3 tests use it.